src/components/Tooltip/index.tsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
-import BaseTooltip from './BaseTooltip';
+import EducationalTooltip from './EducationalTooltip';
+import HoverableTooltip from './HoverableTooltip';
 import type {TooltipExtendedProps} from './types';
 
-function Tooltip({shouldRender = true, children, ...props}: TooltipExtendedProps) {
+function Tooltip({shouldRender = true, shouldRenderWithoutHover, children, ...props}: TooltipExtendedProps) {
     if (!shouldRender) {
         return children;
     }
 
+    if (shouldRenderWithoutHover) {
```

Comment:
So, I thought we would end up without needing this conditional logic 🙁 It might be an oversight on my side, I'll analyze the code in more depth. I thought that we'd know "statically" that our tooltip is either educational, or "standard", hoverable.

Reply:
Okey, so as I see it, 99% of the current `Tooltip` uses could be replaced with `HoverableTooltip`.

If the reason you didn't do this was to avoid a big Git diff (potentially conflict-prone), we could consider changing the naming while keeping the same semantics.

So we could consider renaming `HoverableTooltip` => `Tooltip`, remove `shouldRenderWithoutHover` from `Tooltip` (!), and keep _most_ of the current references. Naming is just naming, it's sometimes a pragmatic thing. You can make the final decision, I don't want to force my way when I don't have a strong opinion.

I can see that, currently, the dynamic switching is used only in `MenuItem.` I still don't understand this fully; I'm analyzing this right now. But, as a no-brainer, you could introduce yet another tooltip component, `MenuItemTooltip`, which does the dynamic switching. Still, I'd like to remove this dynamic switching all together _if possible_.

I hope that my motivation here is clear. While the two kinds of tooltips _look_ similar, they serve different purposes and have different technical capabilities. I would like to make it explicit on the use site which one we use in a specific case.